dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImpl.java

```diff
@@ -112,22 +106,21 @@
     private static final ESIndexAPI esIndexApi = new ESIndexAPI();
     private static final ESMappingAPIImpl mappingAPI = new ESMappingAPIImpl();
 
-    public static final SimpleDateFormat timestampFormatter = new SimpleDateFormat("yyyyMMddHHmmss");
 
     public ContentletIndexAPIImpl() {
         queueApi = APILocator.getReindexQueueAPI();
     }
 
     public synchronized void getRidOfOldIndex() throws DotDataException {
         IndiciesInfo idxs = APILocator.getIndiciesAPI().loadIndicies();
-        if (idxs.working != null)
-            delete(idxs.working);
-        if (idxs.live != null)
-            delete(idxs.live);
-        if (idxs.reindex_working != null)
-            delete(idxs.reindex_working);
-        if (idxs.reindex_live != null)
-            delete(idxs.reindex_live);
+        if (idxs.getWorking() != null)
+            delete(idxs.getWorking());
+        if (idxs.getLive() != null)
+            delete(idxs.getLive());
+        if (idxs.getReindexWorking() != null)
+            delete(idxs.getReindexWorking());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This statement should have braces](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=3712629)